### PR TITLE
desktop macos

### PR DESCRIPTION
**Pull request by kechankrisna**

### CODE REVIEW

**Review by kechankrisna — COMMENTED** [review]
tested

**Review by kechankrisna — COMMENTED** [review]
tested

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.9.0+1 (3rd Jun 2021)
+
+- macos and window support
```
Comment (shilangyu) [inline CHANGELOG.md:5]:
```suggestion
## 0.10.0 (3rd Jun 2021)

- macOS and Windows support
```

File: README.md [hunk 2]
```diff
@@ -82,6 +84,10 @@ Shown below is the full list of attributes which you can specify within your Flu
 
 - `image_path_ios`: The location of the icon image file specific for iOS platform (optional - if not defined then the image_path is used)
 
+- `image_path_macos`: The location of the icon image file specific for MACOS platform (optional - if not defined then the image_path is used)
+
+- `image_path_windows`: The location of the icon image file specific for WINDOWS platform (optional - if not defined then the image_path is used)
+
```
Comment (shilangyu) [inline README.md:90]:
```suggestion
- `image_path_macos`: The location of the icon image file specific for macOS platform (optional - if not defined then the image_path is used)

- `image_path_windows`: The location of the icon image file specific for Windows platform (optional - if not defined then the image_path is used)

```

File: lib/main.dart [hunk 3]
```diff
@@ -108,6 +110,14 @@ Future<void> createIconsFromConfig(Map<String, dynamic> config,
   if (isNeedingNewIOSIcon(config)) {
     ios_launcher_icons.createIcons(config, flavor);
   }
+
+  if (isNeedingNewMACOSIcon(config)) {
+    macos_launcher_icons.createIcons(config, flavor);
+  }
+
+  if (isNeedingNewWINDOWSIcon(config)) {
```
Comment (shilangyu) [inline lib/main.dart:118]: Why `MACOS` and `WINDOWS` are capitalized?

File: pubspec.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: flutter_launcher_icons
 description: A package which simplifies the task of updating your Flutter app's launcher icon.
-version: 0.9.0
+version: 0.9.0+1
```
Comment (shilangyu) [inline pubspec.yaml:3]: This is a new feature, so `minor` version number should be bumped

**Review by shilangyu — COMMENTED** [review]
Didn't check the code itself (I'll leave it to the maintainers), but I left some stylistic comments. Also, there does not seem to be any tests for windows/macos.

**Comment by RatakondalaArun** [comment]
Hi @kechankrisna,

Thanks for this great PR but unfortunately we had many changes since this PR and post that we added support for the remaining platforms so we are closing this PR.

Once again thanks for your contribution ❤️

**Pull request closed without merging**